cmd/poseidon/main.go

```diff
@@ -213,15 +221,15 @@ func main() {
 	}
 	logging.InitializeLogging(config.Config.Logger.Level)
 	initSentry(&config.Config.Sentry, config.Config.Profiling.Enabled)
-	defer shutdownSentry()
 
 	cancel := monitoring.InitializeInfluxDB(&config.Config.InfluxDB)
```

Comment:
Should we rename this occurrence of `cancel` (and in the line below) to avoid confusion with the new one introduced in line 231?